### PR TITLE
KOGITO-5289 Kogito DRL guide

### PR DESCRIPTION
**Pull request by lucamolteni**
cc @tarilabs @mariofusco @jstastny-cz @danielezonca
quickstart: https://github.com/quarkusio/quarkus-quickstarts/pull/911

### CODE REVIEW

**Review by jstastny-cz — COMMENTED** [review]
Just minor wording comments. I still need to verify the steps using the actual quickstart, will update on that shortly.

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 1]
```diff
@@ -0,0 +1,362 @@
...
+<dependency>
+    <groupId>org.kie.kogito</groupId>
+    <artifactId>kogito-quarkus-rules</artifactId>
+</dependency>
+----
+
+== Writing the application
+
+Let's start from the application domain model.
+This application will approve Loan Applications so we have a class with all the details of the wanted Loan
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:93]:
```suggestion
This application will approve Loan Applications so we have a class with all the details of the wanted Loan:
```

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 2]
```diff
@@ -0,0 +1,362 @@
...
+
+    public void setApproved(boolean approved) {
+        this.approved = approved;
+    }
+}
+
+
+----
+
+And another class with the details of the Applicant
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:163]:
```suggestion
And another class with the details of the Applicant:
```

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 3]
```diff
@@ -0,0 +1,362 @@
...
+----
+
+And that's it: REST endpoint to validate Loan Applications will be automatically generated from this Rule Unit.
+
+
+== Running and Using the Application
+
+=== Running in Developer Mode
+
+To run the microservice in dev mode, use `./mvnw clean compile quarkus:dev`.
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:284]:
```suggestion
To run the microservice in dev mode, use `./mvnw clean quarkus:dev`.
```
compile shouldn't be necessary.

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 4]
```diff
@@ -0,0 +1,362 @@
...
+
+=== Running in JVM Mode
+
+When you're done playing with "dev-mode" you can run it as a standard Java application.
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:288]:
```suggestion
When you're done playing with dev mode you can run it as a standard Java application.
```

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 5]
```diff
@@ -0,0 +1,362 @@
...
+After getting a cup of coffee, you'll be able to run this binary directly:
+
+[source,bash]
+----
+ target/kogito-drl-quickstart-1.0.0-SNAPSHOT-runner
+----
+
+== Testing the Application
+
+To test your application, just send request to the service with giving the person as JSON
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:329]:
```suggestion
To test your application, just send a request to the service with giving the person as JSON
```

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 6]
```diff
@@ -0,0 +1,362 @@
...
+          {"id":"ABC10001","amount":2000,"deposit":1000,
+            "applicant":{"age":45,"name":"John"}},
+          {"id":"ABC10002","amount":5000,"deposit":100,
+            "applicant":{"age":25,"name":"Paul"}},
+          {"id":"ABC10015","amount":1000,"deposit":100,
+            "applicant":{"age":12,"name":"George"}}
+]}'
+----
+
+In the response, the list of the approved applications will be returned
```
Comment (gsmet) [inline docs/src/main/asciidoc/kogito-drl.adoc:349]:
```suggestion
In the response, the list of the approved applications will be returned:
```

**Review by gsmet — CHANGES_REQUESTED** [review]
Thanks! Spotted a couple of minor things.

File: docs/src/main/asciidoc/kogito-drl.adoc [hunk 7]
```diff
@@ -0,0 +1,362 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Using Kogito to add rule engine capabilities to an application
+
+include::./attributes.adoc[]
+:extension-status: preview
```
Comment (tarilabs) [inline docs/src/main/asciidoc/kogito-drl.adoc:9]:
```suggestion
```

**Review by tarilabs — COMMENTED** [review]
as spotted by https://github.com/quarkusio/quarkus/pull/18904#issuecomment-891883313

**Comment by gsmet** [comment]
All my comments are minor, I will finalize this one. Please don't push to it, thanks!

**Comment by gsmet** [comment]
Everything fixed, let's wait for CI and merge.

**Review by gsmet — APPROVED** [review]

**Comment by gsmet** [comment]
Thanks!

**Pull request merged**